docs/How-to-add-new-gem-to-omsagent-bundle.md

```diff
@@ -0,0 +1,49 @@
+## How to add a new gem to omsagent install bundle
+
+In order to add a new Ruby gem into OMSAgent bundle when you have the new_gem_name.gem file on you local system you need to follow these steps:
+1. Update the private branch:
+* Clone Microsoft/Build-OMS-Agent-for-Linux github repo
+* Copy the new_gem_name.gem to Build-OMS-Agent-for-Linux//omsagent/source\ext\gems folder.
+* Navigate to Build-OMS-Agent-for-Linux//omsagent/build and put modified commands in buildRuby.sh
+(Example ${RUBY_DESTDIR}/bin/gem install ${BASE_DIR}/source/ext/gems/new_gem_name.gem)
+* Run buildRuby.sh 100
+* It will build the new changes for ruby under intermediate/*/100/Ruby.
+* Run ../installer/scripts/genRubyInstaller.sh.
+* It will create a new file containing changes for ruby.data in /tmp/installBuilder-ruby-data-$USER
+* Diff /tmp/installBuilder-ruby-data-$USER with the rub.data file from Build-OMS-Agent-for-Linux/omsagent/installer/datafiles/.
+* Observe only the new gem related records are added. Replace installer/datafiles/ruby.data with the content of installBuilder-ruby-data-$USER file
+
```

Comment:
I definitely prefer a real-life example, like I commented on above. I understand this because I described it to you. But imagine someone comes along with no clue, you and I have moved on, and the reader has nobody to turn to. Giving a real example that they can look at, with output, would be a lot more helpful.